modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionServiceImpl.java

```diff
@@ -328,6 +328,42 @@ public CPDefinition updateCPDefinition(
 			serviceContext);
 	}
 
+	@Override
+	public CPDefinition updateCPDefinition(
+			long cpDefinitionId, Map<Locale, String> nameMap,
+			Map<Locale, String> shortDescriptionMap,
+			Map<Locale, String> descriptionMap, Map<Locale, String> urlTitleMap,
+			Map<Locale, String> metaTitleMap,
+			Map<Locale, String> metaDescriptionMap,
+			Map<Locale, String> metaKeywordsMap, boolean ignoreSKUCombinations,
+			String ddmStructureKey, boolean published, int displayDateMonth,
+			int displayDateDay, int displayDateYear, int displayDateHour,
+			int displayDateMinute, int expirationDateMonth,
+			int expirationDateDay, int expirationDateYear,
+			int expirationDateHour, int expirationDateMinute,
+			boolean neverExpire, String externalReferenceCode,
+			ServiceContext serviceContext)
+		throws PortalException {
+
+		_checkCommerceCatalogByCPDefinitionId(
+			cpDefinitionId, ActionKeys.UPDATE);
+
+		CPDefinition cpDefinition =
+			cpDefinitionLocalService.getCPDefinition(cpDefinitionId);
```

Comment:
I think this should satisfy SF.

```suggestion
		CPDefinition cpDefinition = cpDefinitionLocalService.getCPDefinition(
			cpDefinitionId);
```